app/javascript/app/pages/national-circumstances/population/distribution-by-age/distribution-by-age-selectors.js

```diff
@@ -0,0 +1,35 @@
+import { createSelector, createStructuredSelector } from 'reselect';
+import isEmpty from 'lodash/isEmpty';
+
+const getDistributionByAgeData = ({ DistributionByAge = {} }) =>
+  isEmpty(DistributionByAge.data) ? null : DistributionByAge.data;
+
+const getData = createSelector(getDistributionByAgeData, data => {
+  if (!data) return null;
+
+  const barChartData = data.data;
+
+  return barChartData;
+});
+
+const getConfig = createSelector(getDistributionByAgeData, data => {
+  if (!data) return null;
+
+  const { config } = data;
+
+  return config;
+});
+
+const getDomain = createSelector(getDistributionByAgeData, data => {
+  if (!data) return null;
+
+  const { domain } = data;
+
+  return domain;
+});
+
+export const getDummyData = createStructuredSelector({
+  data: getData,
+  config: getConfig,
+  domain: getDomain
```

Comment:
if domain and config are already inside the data object why do we need another two selectors?

Reply:
true, I simplified it, check out the changes